src/main/java/io/jenkins/plugins/adoptopenjdk/AdoptOpenJDKInstaller.java

```diff
@@ -56,6 +64,8 @@
  */
 public class AdoptOpenJDKInstaller extends ToolInstaller {
 
+    private static final boolean DISABLE_CACHE = Boolean.getBoolean( AdoptOpenJDKInstaller.class + ".cache.disable" );
```

Comment:
```suggestion
    private static final boolean DISABLE_CACHE = Boolean.getBoolean( AdoptOpenJDKInstaller.class + ".cache.disable" );
```
```suggestion
    private static final boolean DISABLE_CACHE = Boolean.getBoolean( AdoptOpenJDKInstaller.class + ".cache.disable" );
```
```suggestion
    private static boolean DISABLE_CACHE = Boolean.getBoolean( AdoptOpenJDKInstaller.class + ".cache.disable" );
```

So that Groovy scripts can edit it? No strong opinion